data/wanderer dire storm.txt

```diff
@@ -0,0 +1,136 @@
+
+ship "Dire Storm"
+	sprite "ship/dire storm"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 40500000
+		"shields" 54128
+		"hull" 48962
+		"required crew" 72
+		"bunks" 237
+		"mass" 832
+		"drag" 13.9
+		"heat dissipation" 0.5
+		"fuel capacity" 816
```

Comment:
Why the non-round number?

Reply:
Because wanderer ships have ram scoops stock.

Reply:
That doesn't answer the question.

Reply:
Because wanderer ships have stock ram scoops, they benefit from a non-round fuel number because it takes them less time to charge their first ram-fueled jump. If a ship has fuel for N jumps, it can jump its N+1th jump faster, in the same way that a ship with a ramscoop and a scram drive benefits from having 700 fuel over 600 fuel.